src/Components/SmartComponents/HybridInventory/ImmutableDevicesTab/helpers.test.js

```diff
@@ -33,4 +34,46 @@ describe('useOnLoad', () => {
             }
         );
     });
+});
+
+const defaultColumns = [
+    { key: 'updated', renderFunc: jest.fn() },
+    { key: 'system_profile', props: { width: 15 } },
+    { key: 'groups', props: { width: 15 } },
+  ];
+  
+  describe('mergeAppColumns', () => {
+    test('Should replace default OS column to vulnerabilty OS column', () => {
+      const result = mergeAppColumns(defaultColumns);
+  
+      const osColumn = result.find(
+        (column) => column.key === 'os'
+      );
+      expect(osColumn).toEqual(SYSTEMS_EXPOSED_HEADER.find((column) => column.key === 'os'));
```

Comment:
We have to avoid using the implementation code (SYSTEMS_EXPOSED_HEADER) for assertions in tests. If the value changes so the test will automatically assert against updated values - this way we can skip some bugs.

Reply:
Like the same way you use defaultColumns defined in the test, not from the source code

Reply:
This test is intended to verify we are replacing OS column definitions coming from inventory to Vulnerability (SYSTEMS_EXPOSED_HEADER) entirely.